### PR TITLE
openshift installer: add terraform fmt

### PR DESCRIPTION
**Pull request by sallyom**
Don't merge until this merges: https://github.com/openshift/installer/pull/97

### CODE REVIEW

File: cluster/ci/config/prow/config.yaml [hunk 1]
```diff
@@ -2380,6 +2380,45 @@ presubmits:
         args:
           - ./hack/tf-lint.sh
 
+  - name: pull-ci-origin-installer-gotest
```
Comment (stevekuznetsov) [inline cluster/ci/config/prow/config.yaml]: You already have `pull-ci-origin-installer-unit`, why is this necessary?

Comment (sallyom) [reply cluster/ci/config/prow/config.yaml]: yes, I see that now, I see go test is called with ci-operator (https://github.com/openshift/release/blob/master/ci-operator/config/openshift/installer/master.json#L44)  so I've removed that.

**Comment by sallyom** [comment]
@bbguimaraes @stevekuznetsov this should be good to merge now, thanks

**Pull request merged**